src/index.ts

```diff
@@ -90,6 +109,13 @@ if (process.env.NODE_ENV === 'only-ui') {
 app.get('/health', (_: Request, res: Response) => res.send('ok'))
 app.get('/debug', debug)
 
+if (!config.withOathkeeper) { // ExpressJS proxies Kratos public API
+  app.use('/self-service', function(req: Request, res: Response) {
```

Comment:
Oh, I see now how you solved the cookie problem after sign in! Very smart approach, this really decreases deployment complexity a lot for simple use cases :) Now I also understand why the public URL of kratos is no longer required! I think this slipped my radar in my initial review :)